### PR TITLE
Add radial velocity to initial data solves

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

The radial velocity helps control eccentricity. It's the adot0 parameter in SpEC.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by geoffrey4444 — DISMISSED** [review]
Looks good to me! I was wondering why there are accessor functions for angular and radial velocity, when I thought we frowned on those, but I'm fine leaving them or addressing that question later

File: src/PointwiseFunctions/AnalyticData/Xcts/Binary.hpp [hunk 1]
```diff
@@ -215,7 +218,15 @@ class Binary : public ::AnalyticData<3, Registrars> {
     using type = std::unique_ptr<IsolatedObjectBase>;
   };
   struct AngularVelocity {
-    static constexpr Options::String help = "Orbital angular velocity";
+    static constexpr Options::String help =
+        "Orbital angular velocity 'Omega0' about the z-axis. Added to the "
+        "background shift as a term 'Omega0 x r'.";
+    using type = double;
+  };
+  struct RadialVelocity {
```
Comment (kidder) [inline src/PointwiseFunctions/AnalyticData/Xcts/Binary.hpp]: This parameter is not the radial velocity (it is the radial velocity divided by the radius, or the initial time derivative of the scale factor)

Comment (nilsvu) [reply src/PointwiseFunctions/AnalyticData/Xcts/Binary.hpp]: Renamed to `expansion`

**Comment by nilsvu** [comment]
@geoffrey4444 I have no strong opinion on the accessors. I think properties like mass, spin etc that go into a constructor should also be retrievable from the object. Then equality operators etc can use the public state of the object, which is nicer than using private state. For read-only properties this means accessor functions.

**Review by kidder — APPROVED** [review]

**Pull request merged**